scrapli/driver/core/arista_eos/base_driver.py

```diff
@@ -35,7 +35,7 @@
     ),
     "configuration": (
         PrivilegeLevel(
-            pattern=r"^[a-z0-9.\-@()/: ]{1,63}\(config(?!\-s\-)[a-z0-9_.\-@/:]{0,32}\)#\s?$",
+            pattern=r"^[a-z0-9.\-@()/: ]{1,63}\(config(?!\-s\-)[A-Za-z0-9_.\-@/:+]{0,96}\)#\s?$",
```

Comment:
we use case insensitive matching everywhere, so you can ditch the `A-Z` here!

Reply:
Really?  I could have sworn it didn't work when I tried it without the A-Z.  I'll remove it and test again.  I'll need to resubmit anyhow to bring in the other stuff you merged.

Reply:
Its totally possible I missed a place hahaha -- so if you find it let me know and I will fix :D